client/influxdb.go

```diff
@@ -145,6 +145,27 @@ func (c *Client) Ping() (time.Duration, string, error) {
 	return time.Since(now), version, nil
 }
 
+func (c *Client) Dump(db string) (*http.Response, error) {
```

Comment:
Potentially, yes, but it only exposes what you want to expose.  Returning the entire http response is way to much to return.  Also, it's very idiomatic in Go to pass around readers/writers.